src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -567,5 +585,20 @@ private static string GetFwLinkWithCurrentLcidUri(long fwlinkId)
                 fwlinkId,
                 CultureInfo.CurrentUICulture.LCID);
         }
+
+        public static void QueueDoesNotExist(this ILogger logger, string queueName, string queueNameField, string queueAccountUriField)
+        {
+            _queueDoesNotExist(logger, queueName, queueNameField, queueAccountUriField, null);
```

Comment:
Rather than exposing these as parameters, just use nameof(...) here since these should not be configurable.